conversationdemo/conversationdemo/View/LoadingIndicator.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LoadingIndicator.swift
+//  conversationdemo
+//
+//  Created by Eric Giannini on 6/19/18.
+//  Copyright © 2018 Nexmo, Inc. All rights reserved.
+//
+
+import Foundation
+import NVActivityIndicatorView
```

Comment:
You can use MBProgressHUD instead, so that you don't need to configure the activity indicator.